### ISSUE TITLE
Potential product : Recommender System

### ISSUE THREAD

**Issue by philippeTUESIGELEC** [GroceriStar/tensor-something#4]
**Task :** Develop a recommender system.

**Creation date :** 08 / 04 / 2018

**Status :** Paused - No data for now (last update on 14 / 04 2018)

**Assignee :** Philippe Tu

**Comment by philippeTUESIGELEC** [comment]
We need a lot of data coming from users using Groceristar to make this recommender system. 
I think we are not able to produce a recommender system now if we don't have a lot of data from users.

**Comment by atherdon** [comment]
can you specify what exactly data we need to have?

**Comment by philippeTUESIGELEC** [comment]
2 questions : 
- First, you have to tell me _what exactly you would like to recommend to users in the app_. ( Recipes ? Chefs ? ).
- Secondly, when we know what we want to recommend to users. We can select data to work on and to build a recommender system around it. It generally is data coming from the app and produced by users (Example : the recipes input by users, their preferences by asking them when they sign in for the first time, with those data we can already recommend things for the new user).

When the user has chosen recipes he likes, at this point we can recommend better things because we have more data (examples : what he liked, how does he browse what we recommend, yes its possible to know that, how long does he look at a Recipe Page). 
This way, you can constantly recommend relevant recipes to users and this will look like a good app we found nowadays.

Second question here : _Can you provide data from users that used Groceristar_ ? If not, we need users and Data on their use. If yes, please provide them so then I can work on it.

**Comment by atherdon** [comment]
i got a notification. will reply soon

**Comment by atherdon** [comment]
> what exactly you would like to recommend to users in the app

I assume that we can have 2 separated 2.
**ChickenKyiv** => recipes or meal plans(collection of recipes)
**GroceriStar** => ingredients/products

The problem is - we don't have users to generate that data. so maybe ths is the answer to my question. we'll need a lot of time, while this data will be generated. 

* for recipes i think cool to see how Yummly update their search flow in order to generate less results by asking a simple questions about _what do you want to eat_

Btw, i stored for at least 2 month of receipts of my personal purchases. I can move that data into spredsheet. will this data be good for GS option? Please note, that one of future releases of GS related to adding prices and financial tool for families.

**Comment by philippeTUESIGELEC** [comment]
So from what you said. I conclude that we don't have data to work on yet.
I don't think I can work with your personal purchases spreadsheet.
This recommender system is directly linked to your app and its produced Data.
The best way to create something useful for Groceristar and ChickenKyiv is to work with data coming from the apps because the code will be adapted to those Data (no only the content, but also all the different types of data). Working with data from other sources doesn't work because its not adapted to your apps.
I think we can't progress in this task with the current situation for now.

If you have other ideas that needs Data Science, don't hesitate to tell me.

**Comment by atherdon** [comment]
I agree. We'll wait while we have some data from users. Btw, thanks for
researching this topic. it'll be used for a future releases for sure

**Comment by atherdon** [comment]
#11
